backend/fms_core/template_importer/importers/axiom_preparation.py

```diff
@@ -44,8 +51,10 @@ def initialize_data_for_template(self, properties):
     def import_template_inner(self):
         batch_sheet = self.sheets['Axiom Batch']
         batch_df = batch_sheet.dataframe
-
-        self.initialize_data_for_template(properties=batch_df.values[batch_sheet.header_row_nb][self.properties_starting_index:].tolist())
+        properties = batch_df.values[batch_sheet.header_row_nb][self.properties_starting_index:].tolist()
+        # Replace in the list of property types that are modified in the Template header definition
+        properties = [TEMPLATE_PROPERTY_MAPPING[property] if property in TEMPLATE_PROPERTY_MAPPING.keys() else property for property in properties]
```

Comment:
Apparently, property is a built in function in python...
https://docs.python.org/3/library/functions.html#property

Reply:
you are saying i should choose a different name ?

Reply:
Eh depends on your preference ;)